### PR TITLE
docs: bump manual docker build to v0.13.1-beta

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

File: docs/DOCKER.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Docker container, adding the appropriate command-line options as parameters.
 You first need to build the `lnd` docker image:
 
 ```shell
-⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.11.1-beta .
+⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.12.1-beta .
```
Comment (wpaulino) [inline docs/DOCKER.md:32]: Maybe it's worth waiting to update this once `v0.13.0-beta` is officially released?

**Review by carlaKC — COMMENTED** [review]
Can bump to 0.13.0 as suggested, then good to go.

File: docs/DOCKER.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Docker container, adding the appropriate command-line options as parameters.
 You first need to build the `lnd` docker image:
 
 ```shell
-⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.11.1-beta .
+⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.13.1-beta .
```
Comment (carlaKC) [inline docs/DOCKER.md:32]: s/v0.13.1-beta/v0.13.0-beta

**Review by carlaKC — CHANGES_REQUESTED** [review]

**Comment by Roasbeef** [comment]
Will hold off on merging this till the last item in the 0.13.1 milestone has been merged.

**Comment by carlaKC** [comment]
Closing due to inactivity. Feel free to update and reopen!

**Pull request closed without merging**